modules/gdscript/gdscript_parser.cpp

```diff
@@ -4120,7 +4121,15 @@ bool GDScriptParser::export_annotations(const AnnotationNode *p_annotation, Node
 			push_error(vformat(R"("@export_enum" annotation requires a variable of type "int" or "String" but type "%s" was given instead.)", export_type.to_string()), variable);
 			return false;
 		}
-	} else {
+	} else if (p_annotation->name == SNAME("@export_storage")) {
+		use_default_variable_type_check = false; // Can be applied to a variable of any type.
+
+		// Save the info because the compiler uses export info for overwriting member info.
+		variable->export_info = export_type.to_property_info(variable->identifier->name);
+		variable->export_info.usage |= PROPERTY_USAGE_NO_EDITOR;
```

Comment:
Do we expect usage to be something other than NONE here?

Reply:
Yes, `to_property_info()` can set `PROPERTY_USAGE_NIL_IS_VARIANT` or `PROPERTY_USAGE_CLASS_IS_ENUM`.